File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/GettingStarted/PrintFibonacciNumbersTillN.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      // write your code here
      Scanner sc=new Scanner(System.in);
      int n=sc.nextInt();
      int a=0,b=1,c;
      System.out.println(a);
      System.out.println(b);
      for(int x=3;x<=n;x++){
          c=a+b;
          System.out.println(c);
          a=b;
          b=c;
      }
      
      
   }
  }

File: Android/AnjumanHasan_AnjumanHasan_2024cse1157_2/GettingStarted/PrintFibonacciNumbersTillN.cpp
#include <iostream>
using namespace std;
int fib(int n)
{
    if(n==1)
    {
        return 0;
    }
    else if(n==2)
    {
        return 1;
    }
    else 
    return fib(n-1)+fib(n-2);
}
int main()
{
    int n;
    cin >> n;
    
    //write your code here
    for(int i=1;i<=n;i++)
    {
        cout << fib(i) << endl;
    }
}